core/src/main/java/com/workday/autoparse/json/initializers/CollectionInitializerFactory.java

```diff
@@ -39,7 +42,13 @@ public static CollectionInitializer getCollectionInitializerForClass(
             return HashSetInitializer.INSTANCE;
         } else if (LinkedHashSet.class.equals(clazz)) {
             return LinkedHashSetInitializer.INSTANCE;
-        } else {
+        } else if (Map.class.equals(clazz)) {
+            return HashMapInitializer.INSTANCE;
+        } else if (HashMap.class.equals(clazz)) {
+            return HashMapInitializer.INSTANCE;
+        } else if (LinkedHashMap.class.equals(clazz)) {
+            return LinkedHashMapInitializer.INSTANCE;
```

Comment:
Maybe we could just have `Map` here. Allowing the specifics of LinkedHashMap and HashMap seems unnecessary,  IMHO the data class that receives this only should use the interface `Map`

Reply:
These were the supported map types I gleaned from other classes. So I thought it prudent to allow the same types in collections.

Reply:
fair